sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -125,7 +125,7 @@ private[execution] object HashedRelation {
  *  [number of keys]
  *  [size of key] [size of value] [key bytes] [bytes for value]
  */
-private[joins] class UnsafeHashedRelation(
+class UnsafeHashedRelation(
```

Comment:
We need this change?

Reply:
Ah. this is historical code. A question, should we consider the same when
LongHashedRelation.keys.isEmpty
UnsafeHashedRelation.keys.isEmpty
equalTo EmptyHashedRelation
if it does, we need this private[joins] to be removed. I am not quite sure.

Reply:
Ah, I see. it is a trivial change and it looks okay.